ui/app/routes/vault/cluster/access/identity/aliases/index.js

```diff
@@ -38,12 +38,12 @@ export default Route.extend(ListRoute, {
     willTransition(transition) {
       window.scrollTo(0, 0);
       if (!transition || transition.targetName !== this.routeName) {
-        this.store.clearDataset();
+        this.pagination.clearDataset();
```

Comment:
I feel like this reads nicely because it's clear we're manipulating the store with a custom method here (and not calling a built-in `store` method)